nightly_upload.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# nightly_upload: checks remote file for changes and uploads new version if necessary
+# Usage:
+# set scp(ssh) server with user and path
+# set ssh port
+# create env-variable with private key: $ssh_server_key
+# call script: ./nightly_upload.sh -filename
+
```

Comment:
This lacks modifiers to stop the execution:
`set -euxo pipefail`
See:
- https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail

Why not make it all in a single Makefile?